CONTRIBUTORS.md

```diff
@@ -168,6 +168,8 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for details on how to contribute to Cesiu
   - [Mark Aubin](https://github.com/markaubin)
   - [Michael Shantzis](https://github.com/shantzis1962)
   - [Sebastian Fernandez](https://github.com/SebastianPFernandez)
+- [Beijing Xibushijie]
```

Comment:
Do you have a link to your company website by any chance? i.e. https://cesium.com/. If not, there's no need for brackets around the company name.
```suggestion
- Beijing Xibushijie
```